modules/ossm-configuring-jaeger.adoc

```diff
@@ -31,101 +30,148 @@ The default Jaeger parameters specified in the `ServiceMeshControlPlane` are as
-
-|memory
-|Specified in units of bytes (e.g., 200Ki, 50Mi, 5Gi)
-|Memory limit
-|===
-
-.Sample configurations
-[options="header"]
-[cols="l, a, a"]
+[cols="l, a, a, a, a"]
 |===
-| Parameter |Proof of Concept | Minimal Deployment
+|Parameter |Description |Values |Default Value |Examples
 
-|Node count
-|1
-|3
+|tracing:
+  enabled
+|This parameter enables/disables tracing in {ProductShortName}. Jaeger is installed by default.
+|`true`/`false`
+|`true`
+|
 
-|Requests CPU
-|500m
+|ingress:
+  enabled
+|This parameter enables/disables ingress for Jaeger.
+|`true`/`false`
+|`true`
+|
+
+|jaeger
+   template
+|This parameter specifies which Jaeger deployment strategy to use.
+|`all-in-one`/`production`
+|`all-in-one`
+|
+
+|elasticsearch:
+  nodeCount
+|Number of Elasticsearch nodes to create.
+|Integer value.
 |1
+|Proof of concept = 1,
+Minimum deployment =3
 
-|Requests memory
+|requests:
+  cpu
+|Number of central processing units for requests, based on your environment’s configuration.
+|Specified in cores or millicores (for example, 200m, 0.5, 1).
 |1Gi
-|16Gi
+|Proof of concept = 500m,
+Minimum deployment =1
 
-|Limits CPU
+|requests:
+  memory
+|Available memory for requests, based on your environment’s configuration.
+|Specified in bytes (for example, 200Ki, 50Mi, 5Gi).
 |500m
-|1
-
-|Limits memory
-|1Gi
-|16Gi
+|Proof of concept = 1Gi,
+Minimum deployment = 16Gi*
+
+|limits:
+  cpu
+|Limit on number of central processing units, based on your environment’s configuration.
+|Specified in cores or millicores (for example, 200m, 0.5, 1).
+|
+|Proof of concept = 500m,
+Minimum deployment =1
+
+|limits:
+  memory
+|Available memory limit based on your environment’s configuration.
+|Specified in bytes (for example, 200Ki, 50Mi, 5Gi).
+|
+|Proof of concept = 1Gi,
+Minimum deployment = 16Gi*
+
+|
+4+|{asterisk} Each Elasticsearch node can operate with a lower memory setting though this is *not* recommended for production deployments. For production use, you should have no less than 16Gi allocated to each Pod by default, but preferably allocate as much as you can, up to 64Gi per Pod.
 |===
 
-For production use, you should have no less than 16Gi allocated to each Pod by default, but preferably allocate as much as you can, up to 64Gi per Pod.
 
+.Procedure
+
+. Log in to the {product-title} web console as a user with the cluster-admin role.
+
+. Navigate to *Catalogs* -> *Installed Operators*.
+
+. Click the {ProductName} Operator.
+
+. Click the `Istio Service Mesh Control Plane` tab.
+
+. Click the name of your control plane file, for example, `basic-install`.
+
+. Click the `YAML` tab.
+
+. Edit the Jaeger parameters, replacing the default `allinone` template with parameters for the `production` template, modified for your use case.  Ensure that the indentation is correct.
+
+. Click *Save*.
 
+. Click *Reload*.
+{product-title} redeploys Jaeger and creates the Elasticsearch resources based on the specified parameters.
```

Comment:
Doesn't mention about installing the elasticsearch operator first?

Reply:
That's mentioned during the installation steps, elasticsearch is the first operator installed.  In the flow, this chunk of the doc is configuration that happens after you install the operators, but before you deploy the SM Control plane.